### PR TITLE
IssueFixes

### PR DESCRIPTION
**Pull request by shampur**
Auth-fix, Firstrun-fix, help-icon, supported-browsers, graph-fix.

### CODE REVIEW

**Comment by shampur** [comment]
https://github.com/contiv/ccn/issues/212
https://github.com/contiv/ccn/issues/211
https://github.com/contiv/ccn/issues/209
https://github.com/contiv/ccn/issues/205

File: app/menu/menuCtrl.ts [hunk 1]
```diff
@@ -53,21 +54,35 @@ export class MenuComponent implements AfterViewInit{
...
     ngAfterViewInit(){
         jQuery('.ui.dropdown').dropdown({action: 'hide', duration: 100});
         this.authService.firstrunObservable.subscribe((res) => {
             this.firstRun = res;
-        })
+        });
+    }
+
+    help(){
+        window.location.assign("http://contiv.github.io/");
```
Comment (vishal-j) [inline app/menu/menuCtrl.ts]: I think we should open it in a new tab.

File: app/components/directives/settings/networkcreateform.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import {Component, Input, Output, EventEmitter, OnInit, NgZone} from "@angular/core";
```
Comment (vishal-j) [inline app/components/directives/settings/networkcreateform.ts]: If you run first run wizard once and setup network, logout, then remove the first run key from local storage and log back in, and run the wizard again, the data in the network form is pre-populated with the values from the last time. Why is that happening?

File: app/login/login.html [hunk 3]
```diff
@@ -44,6 +44,13 @@ <h1 class="header" style="margin-left:6px; margin-right:auto;">
                     </div>
                 </div>
 
+                <div class="center" style="margin-top: 5px;">
+                    <span class="supportTooltipText">Supported Browsers</span>
+                    <span class="supportTooltip">
+                        <tooltip [width]="200">Google Chrome<br/>Mozilla Firefox<br/>Safari 10 and above<br/></tooltip>
```
Comment (vishal-j) [inline app/login/login.html]: Did you show this to Atousa?

Comment (shampur) [inline app/login/login.html]: Yes, She gave me the design

File: app/components/directives/settings/networkcreateform.ts (hunk 2 above)
Comment (shampur) [inline app/components/directives/settings/networkcreateform.ts]: I think the logout is not clearing the forms that are stored in wizardservice.ts. Will fix it.

File: app/menu/menuCtrl.ts (hunk 1 above)
Comment (shampur) [inline app/menu/menuCtrl.ts]: Fixed

**Pull request merged**